### DISCUSSION TITLE
Question about shuffling before playing

### DISCUSSION THREAD

**Discussion by pappnu** [DeaDBeeF-Player/deadbeef#2884] [Q&A] [answered]
Is it possible to make Deadbeef shuffle the tracks and then play the track that got shuffled as the first one when starting a playlist? Currently if I start a playlist by double clicking it in the playlist browser Deadbeef always plays the first track, as shown in the playlist view, in that playlist even if shuffling is active.

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
Sorry for taking too long with giving any answer, but really I'm not sure whether it's possible...
I can't think of any user action that can do this.
